fdbserver/masterserver.actor.cpp

```diff
@@ -822,9 +832,18 @@ ACTOR Future<Void> recoverFrom( Reference<MasterData> self, Reference<ILogSystem
 			}
 		}
 
+		for (auto cb : provisionalReplyThunks) {
+			cb();
+		}
```

Comment:
I really like this.  Your code for doing this transformation is far cleaner than mine was.  👍